tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -57,3 +60,124 @@ void ROOT::RDF::RunGraphs(std::vector<RResultHandle> handles)
    for (auto &h : uniqueLoops)
       run(h);
 }
+
+
+/// Compute a running mean of events/s.
+double ROOT::RDF::ProgressHelper::EvtPerSec() const {
```

Comment:
Do you think it'd be possible to test some of the functionality - even if it won't be the progress bar itself?

Reply:
Hmm, possible sure. I didn't think about tests much yet, though.